docs/CONTRIBUTING.md

```diff
@@ -302,6 +303,147 @@ GitHub Actions automatically runs these checks on every pull request:
 - Metadata schema validation
 - Standardized README content and formatting conformance
 
+## Adding a Custom Base Image
```

Comment:
optional: This section makes the doc pretty lenghthy, almost doubling it line count.  Perhaps we should consider slimming this down or extracting the contents to a standalone doc